### ISSUE TITLE
`/**/` comments don't work in template code

### ISSUE THREAD

**Issue by DetachHead** [ota-meshi/eslint-plugin-svelte#329]
### Before You File a Bug Report Please Confirm You Have Done The Following...

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have updated to the latest version of the packages.

### What version of ESLint are you using?

8.30.0

### What version of `eslint-plugin-svelte` are you using?

2.14.1

### What did you do?

<details>
<summary>Configuration</summary>
n/a
</details>

```svelte
{foo(/**/)}
```

### What did you expect to happen?

no error, as this is valid javascript

### What actually happened?

```
[2:21]: Parsing error: Unexpected token ;
```

### Link to **GitHub Repo** with Minimal Reproducible Example

https://ota-meshi.github.io/eslint-plugin-svelte/playground/#eJyrVkrOT0lVslKqTsvP19DX0tLXrFWqBQBPdQaV

### Additional comments

related: https://github.com/sveltejs/svelte/issues/7456

however that issue is only for comments outside of the expression (ie. `{foo() /**/}`). comments inside the expression like `{foo(/**/)}` work correctly in the svelte compiler
